Tiltfile

```diff
@@ -1,4 +1,9 @@
+trigger_mode(TRIGGER_MODE_MANUAL)
```

Comment:
What does trigger_mode does?

Reply:
Sets tilt to not automatically rebuild on every file saved, but instead mark it on the UI and adds a button on the UI to manually rebuild when you want it to. You can see a picture of how it looks like [on their docs page](https://docs.tilt.dev/manual_update_control.html)

Reply:
Cool!
Can we set it only for publishing? (as for other services it might be the desired behavior)

Reply:
Setting it per resource is only possible with `k8s_resource` as far as I saw, so I don't think we can use it with docker_compose

I'll remove the change and keep it only locally